packages/shared/src/components/checklist/ChecklistCard.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { ReactElement, useMemo, useState } from 'react';
+import classNames from 'classnames';
+import { Card } from '../cards/Card';
+import MiniCloseIcon from '../icons/MiniClose';
+import { IconSize } from '../Icon';
+import { ChecklistStep } from './ChecklistStep';
+import { ChecklistCardProps } from '../../lib/checklist';
+import { Action } from '../../graphql/actions';
+import { useChecklist } from '../../hooks/useChecklist';
+
+const ChecklistCard = ({
+  className,
+  title,
+  description,
+  steps,
+  onRequestClose,
+}: ChecklistCardProps): ReactElement => {
+  const activeStep = useMemo(
+    () => steps.find((item) => !item.action.completedAt)?.action.type,
+    [steps],
+  );
+  const [checkedStep, setCheckedStep] = useState<string>(activeStep);
+  const { isDone } = useChecklist({ steps });
+
+  const onToggleStep = (action: Action) => {
+    setCheckedStep((currentCheckedStep) => {
+      if (currentCheckedStep === action.type) {
+        return undefined;
+      }
+
+      return action.type;
+    });
+  };
+
+  return (
+    <div className={className}>
+      <Card className="p-0 rounded-14 border-cabbage-40 hover:!border-cabbage-40 w-[340px]">
```

Comment:
This line still have some wrong colors.

Reply:
Also make sure you covered the mobile view I think that might be missing from this PR now.